controller/src/conf/controller.config.properties

```diff
@@ -18,49 +18,52 @@
 #                java -Dcontroller.service.port=5689 ...
 #          which will set the value for that setting to 5689, regardless of what this file contains.
 
+controller.cluster.name=${CLUSTER_NAME}
+controller.container.count=${CONTAINER_COUNT}
+controller.hostMonitor.enable=${HOST_MONITOR_ENABLE}
+controller.rebalance.interval.seconds.min=${CLUSTER_MIN_REBALANCE_INTERVAL}
 
-controller.containerCount=${CONTAINER_COUNT}
```

Comment:
Same here as in the previous comment related to split controller and Segment Store config in different files.

Reply:
https://github.com/pravega/pravega/pull/4713#discussion_r434296932